tests/test_api.py

```diff
@@ -339,26 +365,40 @@ def test_population_404(self):
         self.assertEqual(response.status_code, 404)
 
     def test_population_api(self):
-        # Population for control should be 10 - 1 + 20 - 2 = 27.
-        # Population for variant should be 11 - 5 + 21 - 10 = 17.
+        # Population for control should be:
+        #   11 - 1 = 10
+        #   10 + 21 - 2 = 29
+        #   29 + 31 - 3 = 57
+        # Population for variant should be:
+        #   15 - 5 = 10
+        #   10 + 25 - 10 = 25
+        #   25 + 35 - 15 = 45
         response = self.client.get(self.url)
         self.assertEqual(response.status_code, 200)
         data = response.json()
         self.assertDictEqual(
             data['population']['control'][0],
-            {'window': self.window1[0].isoformat(), 'count': 9}
+            {'window': self.window1[0].isoformat(), 'count': 10}
         )
         self.assertDictEqual(
             data['population']['control'][1],
```

Comment:
Still not sure why I kept getting 'index out of bounds' here.

Reply:
I think it was because the datetime chosen was more than 24 hours away, so this response returned empty data. I made the same mistake then realized what was happening.